### ISSUE TITLE
⌨ `PoolCandidateTable` New application status value has insufficient colour contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#6765]
## ⌨ Accessibility Issue

Our `PoolCandidateTable` component has insufficient colour contrast when the status column is "New application"

### WCAG 2.1 Criteria

[Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI)

## 🦋 Expected Behaviour

"New application" text should use a darker shade of "Tertiary"

## 📋 Steps to Reproduce

1. Open the "Pool Candidate Table" story
3. Confirm the new application colour's text is too light for the white background

## 📸 Screenshot

![Screenshot 2023-05-31 094509](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/e6e57e75-e619-4bd7-b07f-4b200e936a41)

## 🙋‍♀️ Proposed Solution

- [ ] Use darker shade of `tertiary` colour for new application status

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] New application status meets min contrast of 4.5:1

**Comment by tristan-orourke** [comment]
Try .dark, but it may need to be .darker
